brownie/test/plugin.py

```diff
@@ -12,9 +13,23 @@
 from brownie.utils import color
 
 
+def _get_project_path() -> Optional[Path]:
+    if "--brownie-project" not in sys.argv[:-1]:
```

Comment:
Note: This means `--brownie-project=abcd` won't work, which is also a standard way of passing args. But It's probably good enough.

Reply:
Ahh true..  that should be easy enough to support as well.  Good catch!

Reply:
I am wondering, why not just add the flags in pytest_addoption, regardless of whether we are in a project dir or not?
I mean, if the user doesn't want to use brownie, he can just disable the plugin